spec/features/pilot/step_0_understand_service_spec.rb

```diff
@@ -1,18 +1,18 @@
 # frozen_string_literal: true
 
-# Feature: Step 1 - Pilot understands service before engaging
```

Comment:
Ah! I think this probably addresses a lot of the earlier step comments, and I guess it makes sense for step 0 to end with arriving at step 1